backend/app.py

```diff
@@ -0,0 +1,69 @@
+from http import server
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
+import helpers
+
+app = Flask(__name__)
+CORS(app)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///urls.db'
+db = SQLAlchemy(app)
```

Comment:
some reason whenever we try to type in the long url, the long url returns but there is no short url for us to use and type into the search bar at the top of the browser for us to use.